apps/nuxt3-ssr/components/templates/CohortOverview.vue

```diff
@@ -0,0 +1,248 @@
+            },
+            {
+              order: 10,
+              name: "Adult (25-44 years)",
+              code: "25-44 years",
+              parent: {
+                name: "Adult (18+ years)",
+              },
+              mg_insertedBy: "admin",
+              mg_insertedOn: "2022-05-04T14:39:07.451638",
+              mg_updatedBy: "admin",
+              mg_updatedOn: "2022-05-04T14:39:07.451638",
+            },
+            {
+              order: 11,
+              name: "Middle-aged (45-64 years)",
+              code: "45-64 years",
+              parent: {
+                name: "Adult (18+ years)",
+              },
+              mg_insertedBy: "admin",
+              mg_insertedOn: "2022-05-04T14:39:07.451638",
+              mg_updatedBy: "admin",
+              mg_updatedOn: "2022-05-04T14:39:07.451638",
+            },
+            {
+              order: 12,
+              name: "Aged (65+ years)",
+              code: "65+ years",
+              parent: {
+                name: "Adult (18+ years)",
+              },
+              children: [
+                {
+                  order: 13,
+                  name: "Aged (65-79 years)",
+                  code: "65-79 years",
+                  parent: {
+                    name: "Aged (65+ years)",
+                  },
+                  mg_insertedBy: "admin",
+                  mg_insertedOn: "2022-05-04T14:39:07.451638",
+                  mg_updatedBy: "admin",
+                  mg_updatedOn: "2022-05-04T14:39:07.451638",
+                },
+                {
+                  order: 14,
+                  name: "Aged (80+ years)",
+                  code: "80+ years",
+                  parent: {
+                    name: "Aged (65+ years)",
+                  },
+                  mg_insertedBy: "admin",
+                  mg_insertedOn: "2022-05-04T14:39:07.451638",
+                  mg_updatedBy: "admin",
+                  mg_updatedOn: "2022-05-04T14:39:07.451638",
+                },
+              ],
+              mg_insertedBy: "admin",
+              mg_insertedOn: "2022-05-04T14:39:07.451638",
+              mg_updatedBy: "admin",
+              mg_updatedOn: "2022-05-04T14:39:07.451638",
+            },
+          ],
+          mg_insertedBy: "admin",
+          mg_insertedOn: "2022-05-04T14:39:07.451638",
+          mg_updatedBy: "admin",
+          mg_updatedOn: "2022-05-04T14:39:07.451638",
+        },
+      ],
+      mg_insertedBy: "admin",
+      mg_insertedOn: "2022-05-04T14:39:07.451638",
+      mg_updatedBy: "admin",
+      mg_updatedOn: "2022-05-04T14:39:07.451638",
+    },
+  ],
+};
+
+const json = Object.values(data)[0];
+</script>
+
+<template>
+  <AppWrapper>
+    <template #header>
+      <AppHeader />
+    </template>
+
+    <SearchPage>
+      <template #side>
+        <SearchFilter title="Filters">
+          <SearchFilterGroup title="Search in networks" :json="json" />
```

Comment:
In emx2 this should be metadata driven right? i.e based on columnType

I didn't check SearchFilterGroup if that is the case